tests/thread_tests/remote.go

```diff
@@ -142,6 +142,13 @@ func remote_exec(t *testing.T, command string) string {
 
 	t.Logf("[exec-ssh] %s", command)
 
+	// Remote commands that require sudo might ask for the password. Always pass it in. See https://stackoverflow.com/a/11955358
+	if strings.HasPrefix(command, "sudo ") {
+		command = strings.TrimPrefix(command, "sudo ")
+		escapedPassword := strings.ReplaceAll(remotePassword, `"`, `\"`)
+		command = fmt.Sprintf(`echo "%s" | sudo -S %s`, escapedPassword, command)
```

Comment:
Please test if piping the password with an escaped quotation works. If not, I suggest checking and rejecting passwords with quotations.

Reply:
Tested, and it works as expected.

When specifying the remote password variable one needs to escape that too, but that should be expected:
```
REMOTE_PASSWORD="pass\"word" go test
```